dialog/src/main/java/roboy/talk/Verbalizer.java

```diff
@@ -54,23 +54,42 @@ public Action verbalize(Interpretation interpretation){
 
 	public static final RandomList<String> greetings =
 			new RandomList<>("hello","hi","greetings",
-                    // "good morning", // not for the final demo ;)
+                    "good morning", // not for the final demo ;)
                     "howdy",
-                    // "good day", // not for the final demo ;)
-                    "hey", "good evening",
-					"what's up", "greeting to everyone here","hi there people",
-					"hello world","gruse gott","wazup wazup wazup","howdy humans",
-                    // "good day ladies and gentlemen",  // not for the final demo ;)
-                    "good evening ladies and gentlemen",
-                    "hey hey hey you there");
+                     "good day", // not for the final demo ;)
+                    "hey",
+					"good evening",
+					"what's up",
+					"greetings to everyone here",
+					"hi there people",
+					"hello world","gruse gott",
+					"wazup wazup wazup",
+					"howdy humans",
+                     "good day ladies and gentlemen",  // not for the final demo ;)
+                     "good evening ladies and gentlemen",
+                    "hey hey hey you there",
+					"Hey! I was looking for someone to chat.");
+
+	public static final RandomList<String> privateGreetings =
+			new RandomList<>("hello.","hi.","greetings!",
+					"howdy!",
+					"hey, ",
+					"what's up?",
+					"hi there!",
+					"hello world!",
+					"gruse gott!",
+					"servus,",
+					"wazup wazup wazup!!!",
+					"howdy humans?",
+					"hey hey hey you there!",
+					"Hey! I was looking for someone to chat.");
 	
 	private SpeechAction greet(Interpretation interpretation){
 		return new SpeechAction(StatementBuilder.random(greetings));
 	}
 	
 	public static final RandomList<String> farewells = new RandomList<>(
-			"ciao", "goodbye", "cheerio", "bye",
-            "see you", "farewell", "bye-bye");
+			"ciao", "goodbye", "cheerio", "bye", "farewell", "bye-bye");

```

Comment:
Is there a reason why see you is removed?

Reply:
yes, cause we want to answer "where can i see you?" which would never get through